server/src/main/java/io/spine/server/model/Model.java

```diff
@@ -83,14 +86,35 @@ void clear() {
         checkNotNull(cls);
         ModelClass<?> modelClass = classes.get(cls);
         if (modelClass == null) {
-            modelClass = AggregateClass.of(cls);
+            modelClass = new AggregateClass<>(cls);
             checkDuplicates((CommandHandlingClass) modelClass);
             classes.put(cls, modelClass);
         }
         return (AggregateClass<?>) modelClass;
     }
 
     /**
+     * Obtains an instance of aggregate part class information.'
+     *
+     * <p>If the passed class was not added to the model before, it would be added as the result of
```

Comment:
I would say
```
If the passed class is not added to the model before, it will be added as the result of this method call.
```
instead. It's a bit simpler to comprehend, as to me.